### ISSUE TITLE
AVT 1 - Toggle and Untoggle have DAP violations when disabled

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4543]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="918" alt="Screen Shot 2019-11-01 at 4 40 22 PM" src="https://user-images.githubusercontent.com/21676914/68058320-b01e2300-fcc6-11e9-8cf2-ed56278c77a0.png">

Note: Missing disabled flag or aria-disabled="true" which is causing the DAP violation.

**Comment by snidersd** [comment]
Same violations are also present for ToggleSmall.

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
